src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -63,6 +63,9 @@ public function getProperties($class, array $context = array())
         foreach ($reflectionClass->getMethods(\ReflectionMethod::IS_PUBLIC) as $reflectionMethod) {
             $propertyName = $this->getPropertyName($reflectionMethod->name);
             if ($propertyName) {
+                if (!isset($properties[$propertyName])) {
+                    $propertyName = lcfirst($propertyName);
```

Comment:
You should use this regex instead:

```
// if more than two uppercase characters don't lcfirst
return preg_match('/[A-Z]{2,}$/', $name) ? $name : lcfirst($name);
```

(First proposer by @soyuka in https://github.com/dunglas/DunglasApiBundle/pull/260/files)

It allows to handle propertly methods like `getIBAN()`.